packages/core/src/js/feedback/icons.ts

```diff
@@ -0,0 +1,2 @@
y1XnuxYbbUR/OmtEZSfd1iUl2L4fDLvXACl1oqr2TTaLbR9kfQ9s34vGq0Mn7ux8KiqRCCBxvrUmdtz+++sr9/xet++sO2yycwpgAMN9ezsWeL2Cj2PtQDq8l2Dv/c7deYl9C3OpfudttuufZwvfsqZkqWW63k8uPEeHvTrHm/EbH4j+FBsmSjFS6I08iLDopXndBQm7qsO5u2ZrlxCwiBopcAAfE9Em5at2h3925OYUzmTr0HZ+URWFr628ba1ruYhzA4X00KQAD6ypqJhzGZz2Ct+CkQgZyuLuNywJo9dKWVh7ApkXqPrn7Sr30EKAD7WGj/qjvX/qfC1u1zMT5yr87OWnkI8fdcsjZ1mxApLccvdOZXSd8oAJWk7UJb1iq9zt62S5VZOBtrSl52waQ3JpCHECtNbmyqE88nZ7aMOtDpTeO0WioBCkCppDQr15nreAxJPI7C3PNfaObaW9yx7hSpmJFG0pFr8IF3o6lvaZVvSiVAASiVlIblrAw+SChxDrLHXgYh2HuHRjdXcTkwAcf+nbgk+AOWGc/Szb8o+0MBCEHvI3vsYizmmotV5k9rHY4UJ2GYcHky0bpAaz8j5BwFICSdjWSSfZls+hSsQP0MbhnaelZ8JVDgGmCyMORirC58tH56C3NRVgL6GG1QAMaAE8CPFJ4gc6csmClcErTp7D+E4OyaiXIl8xD620sUAH/5e9J6pr+9s7NXHGs9TBKXBRqnfZeHCYk8hHWti+YcMucgT2DQ6JgESp4KPKYVfqghgfRgJovbcInUL5H6HVOJ5Ts1dHLIJfh2aXzygZcmJ6d0dTG0fvEMILRd+0Zgnbn0nzftks1M9hLyjrYZHgXAJrggVbOeJ28le0FK6tMweejFIPlOX70lQAHwlq9W1q2Velvzg0dDBB7UyjE64xsBCoBv6P1p2Fqzn+ltuwipqy/E7UI8Fo5blAlQACLa+0hd/dNC3jwaqwt/G1EEDBsEKAAR/hqsWtO+DunfPoTk9Z/EIKH16CpuESNAAYhYhxcLt6s3fbcYFNadgv8t9jn3hZcABSC8fVtWZJm1bT24U/A+XBJ8ERUHy6rMwoElQAEIbNd54ngBlwQLhVk4BlOJV3rSAo1qRYACoFV36OFMJtfRgTyE78Jdgm/gjED759vpQS2YXlAAgtlvnnv9Zh7C65B+7GSIQM7zBtmALwQoAL5gD06jWGb8zFAeQlMtCo7XPnqqgjV+QgHw8bsSlKaH8hDm2j4mlHmW1nkIdQAq1Us6uFGqDxSAUkmxHB4e2/64lYcQIvAocRQnoAbzy4p/oudeCoCe/aKtV1YeQkwl/idhigU65yH0AyAGTV/qWrM840fbdtukANglF/F6mVz6PmT1wcIi8ceIo9gbvhTqfryBLgZnowAEp6+08zTTm+7H36lYWHSNznkIKwJOiZ2Dg4VvV6QtFxuhALgIM6KmFBYWfWvQVHhStPL9ac5+9YEp1O2r1y4LXK4FCoBf35iQtfu3vrauTK88zlTiNr3zELoPHnMlnurKtn3VfcveW6QAeM84Qi2kB7uy6ZuEyL8XlwSrIxL4H3cJ+RHEqnHy1dF7ggIwOht+YpNAJrvshU0DogVPM/4eZhEGalCsnJDxy7840ytO6+1Nby6nnk5lKQA69UaIfLHyEOK0+KpCXh2DsEJ1pwCKtrIg1Bl4SOtlQqQDvXKSacFDdNDpGAqSjvwVfp0yO5FqwHPCz1dSvU9K2YBfz8OQDrxaR5+H+2QthsLtTusXvgePNl2Ks5qfY3r0s3gfijMbCsDw3uZrzwiszqW7YfxWzxqgYVsEeAlgCxsrkUA4CFAAwtGPjIIEbBGgANjCxkokEA4CFIBw9COjIAFbBCgAtrCxEgmEgwAFIBz9yChIwBYBCoAtbKxEAuEgQAEIRz8yChKwRYACYAsbK5FAOAhQAMLRj4yCBGwRoADYwsZKJBAOAhSAcPQjoyABWwQoALawsRIJhIMABSAc/cgoSMAWAQqALWysRALhIEABCEc/MgoSsEWAAmALGyuRQDgIUADC0Y+MggRsEaAA2MLGSiQQDgIUgHD0I6MgAVsEKAC2sLESCYSDAAUgHP3IKEjAFgEKgC1srEQC4SBAAQhHPzIKErBFgAJgCxsrkUA4CFAAwtGPjIIEbBGgANjCxkokEA4CFIBw9COjIAFbBCgAtrCxEgmEgwAFIBz9yChIwBYBCoAtbKxEAuEgQAEIRz8yChKwRYACYAsbK5FAOAhQAMLRj4yCBGwRoADYwsZKJBAOAhSAcPQjoyABWwQoALawsRIJhIMABSAc/cgoSMAWAQqALWysRALhIBDXNgwlLm+qbf2Q3/5JKU702we2TwJeEdBXAKSYL4X0Km7aJQESAAFeAvBrQAIRJkABiHDnM3QSoADwO0ACESZAAYhw5zN0EqAA8DtAAhEmQAGIcOczdBKgAPA7QAIRJkABiHDnM3QSoADwO0ACESZAAYhw5zN0ErAtAEqoXcTnjICUsuDMAmuTgDMCtgVAKPGSs6ZZGwTWkwIJ+EnAtgAoIZb76XhI2l4WkjgYRkAJ2BYAQ5lPBjRmfdxW6nF9nKEnUSRgWwAyuY4OnAWsjCI0l2IeHByQP3fJFs2QgC0CtgXAag2r9b9iq1VWEkqJu1evS79KFCTgJwFHApDpTT+Ab/Kf/AwgiG3j4H/N3L391iD6Tp/DRcCRAACF2iF2X4D/a8KFxdNo8tJU53e/2P2ap63QOAmUQMCpAIhsduWGQWWegduCG0poL+pF8sIUH8/0tf0h6iAYvx4EXEu6V3/E3BnVNVWPwWCrHqHp5YV12m/98vPg16tfou5NzC0AG7du2HLY61PvNafEtgkpWyAEE92yHXA7gzj4v49r/nO71qzoDHgsdD9kBFw7AxjOZcaMGQccFD/0LEx1PR0NpPDZTIjCpOFlQvtaiYKQar0Scpkw1eP53fJnHO0PbW8HPrD/BxshrefcOuZIAAAAAElFTkSuQmCC';
```

Comment:
i: could you check if the below image works fine? if so, it will decrease the added size of 10KB to 2.6KB
```suggestion
  'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAQAAAAEACAMAAABrrFhUAAAAk1BMVEUAAAAqIDEqIjMmIDArIjIrIjMoHjErITMrIjMqITIqITIqITIqIjIqITIqITIhHi4qITIqITIeAConHi8rITMqITIqITIcHCopITIpIDEpIDIlGC8qITIqITIpIDIpHjAqITIqIDIqIDEqIDAqITInHjAqIDIqIDEqITIqITIqIDIiFCgpIDEqITIoHS8qIDIrIjMN1S0HAAAAMHRSTlMAhnknzMQy1zyF6/H55PYQ0rIGGPudjAmUTEQUv7p2K6lmWDZxHV9S36J+DD7bI208pRBPAAAGrUlEQVR42uzd6XLaMBSG4Y/YhuAVbAwx+xp2eu7/6tq0nfaHIAizSNbR+5vJJM8EG45lGfrm7EZRSNL5bvcAg1oM6fZaBQxp/0HlylOY0PuRyjbco/L1unRHbVS9RYPuaoxKlzY7dF9xhgpXb9Hd1VDddiHdX4SqNnHpIU1QzcYhPaY5qpjj0aNaooL9iOlhvaFynYZEnAFmAXEGWI2IOAN8BsQZIGsTcQaY+0SMAQZT+hVfgLVPX3EFSLb0J6YARUR/YwmQ5PQvjgD9DcmWJ7hUraoANwy9owIwDuCGofc2gXEANwy9/TVgHMANQ+/pAMYBpLUOSebPAeMAbhh6tzOYByA/9A4+AeMAbhh6j1YwD0B+6B3MAOMAbhh6D09QAdCvtT33ecUkWfwDeD1Ab+yTHnknKABYHEmPwjGgAKAISY/cCVQAvGvy94c7QAVAEpEWtepQA/BBOtRppngYwDaDfAMt3gCNBVAeQKzxUciC/iAN6vZwJ4BYsC0gU5uUd3wHHgAgFu0yXE39IfCjhwcBiHWmE1yJFLfpA+UBJJpOtAbI93gwgFieaQsQFcDzAShYagqwTfAUADF3oiGAvwZeBUDxUjuA6QBPBBAbDbQC8OfAawFoU9cIoJ3h5QAUz3UBCD4BBQBESz0ARisoAqCmBgDBDFAGQB/KAaYrqASgrlKAVtcB1ALQThJgXX94px4A5QD0KQfgQP/KAYQH5gDkZ8wByOMOQE3DAfJZHtF3dd7NBngDsKh9Z9BIjQf41dyjizU5AACHFl0odFgAADOfztdmAoBsROfrMwEAxh06l8cGAEVM5+qzAcDCpzMN+QBgEtCZ6nwAcAhJbMsIAHMSixNGAPggsRkngF6DhDxOAKiHJJRxAjj3JvjBCmAgngvbrACwE88DKSuARPxIfGAFgFx8JS+AungQ4AUAYUAUMQMQ79pNeAH0hZe+8wJIY+FKKS8AuMLFcmYAubBihBnAUliwwgxgLQwGmQEIpwGXGcBCWLXEDGAiXCZmBlDn/h9w4H4MKIS5KDOAmfB9mBmA8HUwZwYgLBaoMQOIhItDvAAccZEAL4Cl8NIVLwDhEBDwmgmKeyV4vADG4v0jvACO4m0hrAAKEspYAYjrZl1WV4c/SajGCWAfkdCCE0BOQhtOa4TWJNZlBLDySczhA5A0SMzjs1I0HdKZ5mwAem06U8RmtXji0bmWXACcBp3L7zEBuPQcpyWPW2ayKZ1vk3IASMcBXWjN4La5/exIlxqZf+fopBvQxeKT2QD7fvdI3zU2++7xgK40NHz/gGsFK+YABfMtNJrMd5GZQgpgE4RR47F50/FEPYDXU7qXmFsoBmglKAGg316KZQEaGcoBaLWZanmAVqZ8O72vtokiAC/RYEPFr6K+EoB2T48tNb/66L0eoKnRrrJEjcWLAYK1VvsKE3V2LwVwHd12lhZ+pzIA8toa7i1OFC9fBODW9dxdnmi4egGAP8M3dahMemwyLQUQ1BJ8l/KnrLUHzwQIatd+vEeq84unARyXCa7VJPXl+2cAxNM+JJqQBm0Ojwbwp+se5BqRDnXTBwK0xnXIV++QDrXqT7k2WJmnjRGFb88CqMgD94i8kyIAjEmP4pkiALy3SI9GmRoAoL/dkA75czUAXyUnx3HpfHPnWjmdz/nTW0iSTRMFAFc/GL+X/mVuf/J+1DcSAGn3hoGhiQDAYSM/MDQSAPv8hqfwmggArH35JwQaCYBBW35gaCQA8BnIDwyNBMBqKD8wNBIAWMbyA0MjATBx5QeGRgIgbXbkB4YmAgCLI91flQHQ+z+H4QkA9CP6E1cAJFP6HVsAYO7TrxgD/HkMEGcAYBYzB8DJYw4AvIXMAVBvMQdA2mUOABw2zAHEgWH5xpUEAAqfHtO8ogDCwLBsk6oCCAPDckWoLoAwMCxTt8oAwsDw9uJVtQHguPeeBCsOIAwMb2uEygMAiwaVbZiYAFB+YJinMAIAWAzp9loFYAoA4OxGUUjS+W73AJgEUCILYAEsgAWwABbAAlgAC2ABLIAFsAAWwAJYAAtgASyABbAAFsACWAALYAEsgAWwABbAAlgAC2ABLIAFsAAWwAKUANg2r+WZDSCRBbAAFsAC/GyfXlIYhIIgitYgGHhoiMmDSPyAgqKz2v/qxIkL0JoU9NlB3+6OABEgAkSACBABnAP0FMvwslGsgpeBYjW8FBQbYeZNqbKFmYlSDex0FEp2BwB8V8rkPwwViSJ5gaX5QYlkuf/DZ0i8rWwM///0m7bni5flqh5dxt8BPG/wrQSMUX8AAAAASUVORK5CYII=';
```

Reply:
Thank you @lucas-zimerman 🙇 
The provided smaller image works fine 🎉